_posts/2022-04-24-sr2022-knockouts.md

```diff
@@ -0,0 +1,117 @@
+---
+title: TEAM 1st wins Student Robotics Competition 2022!
+image: image
+image_alt: Alt text for the image
+description: SR2022 came to an exciting conclusion with a victory for {TEAM 1st}
+---
+
+That’s it for Student Robotics 2022! {TEAM 1st} took the crown in an intense
+showdown between {TEAM 2}, {TEAM 3}, and {TEAM 4}. It was fantastic to finally
+see robots in-person again after two years of being virtual. The ingenuity and
+engineering displayed by all our teams was incredible and we're delighted with
+the performances we saw over the weekend.
+
+## The Game: This Way Up
+
+This Way Up challenged teams to collect the most tin cans from the arena and
+ensure that they were the right way up for maximum points. When in a team’s
+scoring zone, correctly oriented cans were worth 3 points, upside down cans were
+worth one point, and cans lying on their side were worth nothing. However, not
```

Comment:
Probably should change this to be 3 and 1 points or three and one points so it's consistent